### ISSUE TITLE
Labeling document types

### ISSUE THREAD

**Issue by rajkiranrbala** [cloudfoundry-incubator/cf-abacus#279]
Aggregator produces three documents
1. Org Aggregated Doc
2. Consumer Aggregated Doc
3. Consumer Aggregated Doc at Instance Level

Though the type of doc can be inferred from the document structure/keys, can there be an explicit field thats states the type of the document?

**Comment by jsdelfino** [comment]
These docs have different structures so I'm not sure why you'd need an additional flag or doc/object type field to tell you what structures they have. Javascript objects and JSON typically do not carry an object type label. ES6 introduces classes to Javascript but I don't believe that the class name gets serialized in any way in JSON.

**Comment by jsdelfino** [comment]
Please reopen if I mis-understood this issue.

**Comment by rajkiranrbala** [comment]
I am developing an app the extends the pipeline after aggregator. My application expects only org aggregated document of the 3 documents produced by the aggregator. When the aggregator produced 2 documents, I inferred based on the structure. When I pulled abacus changes and ran the application with it, the app started crashing because of the 3rd document that was produced by the aggregator. So I thought adding a field that identifies the document type would help in picking the right document.  This can be solved in a different way if  issue https://github.com/cloudfoundry-incubator/cf-abacus/issues/278 is addressed
